### PR TITLE
Parser - #29 split command

### PR DESCRIPTION
**Pull request by osvegn**
Pull request link to this issue: #29

### CODE REVIEW

**Review by LucasTesnier — APPROVED** [review]
NTR

File: server/src/parser/my_str_to_word_array.c [hunk 1]
```diff
@@ -0,0 +1,81 @@
+/*
+** EPITECH PROJECT, 2022
+** my_teams
+** File description:
+** my_str_to_word_array
+*/
+
```
Comment (Cavonstavant) [inline server/src/parser/my_str_to_word_array.c:7]: missing /// \file comment

**Review by Cavonstavant — CHANGES_REQUESTED** [review]

File: server/src/parser/my_str_to_word_array.c (hunk 1 above)
Comment (osvegn) [reply server/src/parser/my_str_to_word_array.c:7]: needed in .c files ? not only on .h files ?

Comment (LucasTesnier) [reply server/src/parser/my_str_to_word_array.c:7]: Needed on all files

**Pull request merged**